src/core/util/media.js

```diff
@@ -74,7 +74,7 @@ angular.module('material.core')
  *
  *  See Material Design's <a href="https://material.google.com/layout/responsive-ui.html">Layout - Adaptive UI</a> for more details.
  *
- *  <a href="https://www.google.com/design/spec/layout/adaptive-ui.html">
```

Comment:
Ouch this is actually a 404 now! The new URL looks good.

Reply:
:chicken: